Resources/public/js/views/fields/ez-country-editview.js

```diff
@@ -0,0 +1,225 @@
+                isMultiple: this.get('isMultiple'),
+                resultTextLocator: function (sourceElement) {
+                    return sourceElement.Name;
+                },
+                resultAttributesFormatter: function (sourceElement) {
+                    return {
+                        id: sourceElement.Alpha2,
+                        text: sourceElement.Name,
+                    };
+                },
+            });
+        },
+
+        _addSelection: function (text, id) {
+            var values = this.get('values');
+
+            values.push({text: text, id: id});
+            this._set('values', values, {"added": text});
+        },
+
+        _removeSelection: function (text, node) {
+            var values = this.get('values');
+
+            values = Y.Array.reject(values, function (val) {
+                return (val.text === text);
+            });
+            this._selectionFilter.unselect(text);
+            this._set('values', values, {
+                "removed": text,
+                "node": node || null
+            });
+        },
+
+        _getSource: function () {
+            var countriesArray = [];
+
+            Y.Object.each(this.get('config'), function (country) {
+                countriesArray.push(country);
+            });
+            return countriesArray;
+        },
+
+        _getSelectedTextValues: function () {
+            var field = this.get('field'),
+                fieldDefinition = this.get('fieldDefinition'),
+                valueIndexes = [],
+                config,
+                that =  this,
+                res = [];
+
+            if ( field && field.fieldValue ) {
+                valueIndexes = field.fieldValue;
+            }
+            if ( fieldDefinition && this.get('config') ) {
+                config = this.get('config');
+            }
+
+            Y.Array.each(valueIndexes, function (index) {
+                res.push({text: that._getCountryName(index), id: index});
+            });
+            return res;
+        },
+
+        _variables: function () {
+            var def = this.get('fieldDefinition'),
+                selected = [];
+
+            Y.Array.each(this.get('values'), function(value) {
+                selected.push(value.text);
+            });
+
+            return {
+                "isRequired": def.isRequired,
+                "selected": selected,
+                "isMultiple": this.get('isMultiple'),
+            };
+        },
+        _getFieldValue: function () {
+            var res = [];
+
+            Y.Array.each(this.get('values'), function(value) {
+                res.push(value.id);
+            });
+            return res;
+        }
+    },  {
+        ATTRS: {
+            /**
+             * The text and id values of the selected options
+             *
+             * @attribute values
+             * @readonly
+             * @default []
+             * @type Array
+             */
+            values: {
```

Comment:
you don't need to redefine the attribute but it's good to "override" the comment since the content of the attribute is specific to the country edit view.